test/models/match_test.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class MatchTest < ActiveSupport::TestCase
```

Comment:
I don't use minitest so not sure what best practices are, please comment away

Reply:
Nor do I but this looks great to me!